soyuz_app/static/sass/partials/_app.scss

```diff
@@ -12,35 +12,38 @@ body {
 .btn-light {
   color: red;
 }
+.btn-primary {
```

Comment:
in general we should avoid applying CSS to base bootstrap styles unless we know 100% for sure that it will apply in all cases.

top and bottom margin doesn't seem to be applicable everywhere.

besides creating another class to apply to the button, another option is to use SASS to apply this style to this class nested in another specific class, that way the btn-primary style only applies in those particular cases